docs/ui-configuration.md

```diff
@@ -0,0 +1,25 @@
+## UI Configuration
+
+The UI configuration supports the following params:
+
+- As defined in type [UIOptionsObject](https://github.com/kaltura/playkit-js-ui/blob/master/docs/configuration.md)
```

Comment:
add also line for link to UI guides: https://github.com/kaltura/playkit-js-ui/blob/master/docs/guides.md